### ISSUE TITLE
New installation

### ISSUE THREAD

**Issue by Burz-HP** [Requarks/wiki#1868]
**Question**
Hello,

Currently in the market for a new wiki, have installed Wiki.js to give it a try and after installation I am seeing 2 issues;

1. On page refresh, I am seeing an error: An unexpected error has occured
This is primarily over the search bar.

2. In attached image, see that all items in menu's and on pages have their name followed by .ITEMTYPE, for example; navigation.title

How do I change this so it just says; Navigation, it's a bit of an eye sore :)

Installation is in a closed environment, using the sideload method to import data .

Thanks for you help.

**Host Info (please complete the following information):**
OS: Windows Server 2019
Wiki.js version: 2.3.81
Database engine: postgres 12.2.4

![2020-05-11_16h26_52](https://user-images.githubusercontent.com/65180277/81580455-5f8a2f80-93a5-11ea-9b40-425223f043c1.png)

**Comment by NGPixel** [comment]
Open the developer tools, in the Network tab, do you see any errors? Looks like the localization strings fail to load.

**Comment by supersasho** [comment]
Seems like you're about right with the locale missing. I did a **new container** through **docker**, used the latest image **2.3.81** and have the exact same issue.

Another wiki container that I update regularly for months is fine though.

In **logs** I can't see anything related, other than a **warning**:
**"2020-05-13T14:29:47.109Z [MASTER] warn: EISDIR: illegal operation on a directory, read"**

![image](https://user-images.githubusercontent.com/1937918/81853690-0c5add00-955d-11ea-8875-546285698af0.png)

**Comment by supersasho** [comment]
So after some digging I've stumbled upon the error message below. If I can provide more details let me know. Thank you.

```yaml
[{errors: [{message: "Invalid locale or namespace", locations: [{line: 3, column: 5}],…}],…},…]
0: {errors: [{message: "Invalid locale or namespace", locations: [{line: 3, column: 5}],…}],…}
errors: [{message: "Invalid locale or namespace", locations: [{line: 3, column: 5}],…}]
0: {message: "Invalid locale or namespace", locations: [{line: 3, column: 5}],…}
message: "Invalid locale or namespace"
locations: [{line: 3, column: 5}]
0: {line: 3, column: 5}
line: 3
column: 5
path: ["localization", "translations"]
0: "localization"
1: "translations"
extensions: {code: "INTERNAL_SERVER_ERROR", exception: {stacktrace: ["Error: Invalid locale or namespace",…]}}
code: "INTERNAL_SERVER_ERROR"
exception: {stacktrace: ["Error: Invalid locale or namespace",…]}
stacktrace: ["Error: Invalid locale or namespace",…]
0: "Error: Invalid locale or namespace"
1: "    at Object.getByNamespace (/wiki/server/core/localization.js:56:13)"
2: "    at translations (/wiki/server/graph/resolvers/localization.js:36:24)"
3: "    at field.resolve (/wiki/node_modules/graphql-extensions/dist/index.js:133:26)"
4: "    at resolveFieldValueOrError (/wiki/node_modules/graphql/execution/execute.js:467:18)"
5: "    at resolveField (/wiki/node_modules/graphql/execution/execute.js:434:16)"
6: "    at executeFields (/wiki/node_modules/graphql/execution/execute.js:275:18)"
7: "    at collectAndExecuteSubfields (/wiki/node_modules/graphql/execution/execute.js:713:10)"
8: "    at completeObjectValue (/wiki/node_modules/graphql/execution/execute.js:703:10)"
9: "    at completeValue (/wiki/node_modules/graphql/execution/execute.js:591:12)"
10: "    at /wiki/node_modules/graphql/execution/execute.js:492:16"
11: "    at runMicrotasks (<anonymous>)"
12: "    at processTicksAndRejections (internal/process/task_queues.js:97:5)"
13: "    at async Promise.all (index 0)"
data: {localization: {translations: null, __typename: "LocalizationQuery"}}
localization: {translations: null, __typename: "LocalizationQuery"}
translations: null
__typename: "LocalizationQuery"
```

**Comment by winkelement** [comment]
So you set config option `offline: true` and enabled sideloading?
What are the permissions for your `data/sideload` folder?

**Comment by supersasho** [comment]
Well I didn't set any config option at all. It was brand new docker image pulled and run. No sideload folder in the data folder can be found:

```bash
bash-5.0$ ls -l
total 20
drwxr-xr-x    2 node     node         12288 May 13 14:31 cache
drwxr-xr-x    3 node     node            18 May 13 14:27 content
drwxr-xr-x   12 node     node          4096 May 13 14:50 repo
drwxr-xr-x    2 node     node             6 May 13 13:19 uploads
bash-5.0$
```

**Comment by winkelement** [comment]
Check out the docs for this 
https://docs.requarks.io/install/config#offline-mode
https://docs.requarks.io/install/sideload

Also you might want to check how to mount a config file https://docs.requarks.io/install/docker#alternative-mount-the-config-file

**Comment by supersasho** [comment]
Thank you that helped. I didn't know that the locale is being downloaded from another source.

**Comment by gerroon** [comment]
I did sideloading and enabled true and Is till have some of the text misrepresented


See the image please

https://i.imgur.com/95nMjBv.png


I am using Docker composer Wiki.js v2 image.

**Comment by thehoneybug** [comment]
I had the same issue when installing on windows server 2012. 

On loading the application in the browser, every control was showing name with a dot seperating two words. Also a error was popping up at the center of the page. 
i tried to look for error logs but did not find any when I found that the error logging happens at node level and that it writes to the console. Since I was using the chrome browser I navigated to the console and bam. graph ql error: invalid locale or namespace was being thrown. 
I had set the offline parameter in the config.yml file to true on configuration. 
Seems that the application could not download content needed to get going. 
I set it back to false and everything sorted out!
